pkg/apis/v1/nodeclaim_validation.go

```diff
@@ -75,11 +72,10 @@ func (in *NodeClaim) Validate(_ context.Context) (errs *apis.FieldError) {
 	)
 }
 
-func (in *NodeClaimSpec) validate() (errs *apis.FieldError) {
+func (in *NodeClaimTemplateSpec) validate() (errs *apis.FieldError) {
```

Comment:
Why did we differentiate between NodeClaimSpec and NodeClaimTemplateSpec now?

Reply:
This would be to keep all the fields that are shared between NodeClaim and NodePool

Reply:
Removed from this PR